apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/tpsmessagingservice/TpsMessagingConsumer.java

```diff
@@ -95,19 +96,18 @@ public static String tilfeldigUtlandskBankkonto(String landkode) {
     }
 
     @Timed(name = "providers", tags = {"operation", "tps_messaging_createUtenlandskBankkonto"})
-    public List<TpsMeldingResponseDTO> sendUtenlandskBankkontoRequest(String ident, List<String> miljoer, Object body) {
+    public List<TpsMeldingResponseDTO> sendUtenlandskBankkontoRequest(String ident, List<String> miljoer, BankkontonrUtlandDTO body) {
 
-        if ((body instanceof BankkontonrUtlandDTO bankkontoUtland)
-                && (null != bankkontoUtland.getTilfeldigKontonummer())
-                && bankkontoUtland.getTilfeldigKontonummer()) {
-            body = bankkontoUtland.withKontonummer(tilfeldigUtlandskBankkonto(bankkontoUtland.getLandkode()));
+        if ((null != body.getTilfeldigKontonummer())
```

Comment:
Ville brukt `nonNull()` i stedet for `null !=`

Reply:
👍